### PR TITLE
docs(i18n): Update PL localisation

### PR DESCRIPTION
**Pull request by zwiastunsw**
update to new version

### CODE REVIEW

File: locales/pl.json [hunk 1]
```diff
@@ -545,9 +587,16 @@
...
+    "color-contrast": {
+      "pass": {
+        "default": "Element ma wystarczający kontrast kolorów  ${data.contrastRatio}",
+        "hidden": "Element jest ukryty"
+      },
+      "fail": {
+        "default": "Element ma niewystarczający kontrast kolorów: ${data.contrastRatio} (foreground color: ${data.fgColor}, background color: ${data.bgColor}, font size: ${data.fontSize}, font weight: ${data.fontWeight}). Oczekiwany współczynnik kontrastu: ${data.expectedContrastRatio}",
+        "fgOnShadowColor": "Element ma niewystarczający kontrast kolorów: ${data.contrastRatio} pomiędzy kolorem pierwszego planu a kolorem cienia (foreground color: ${data.fgColor}, text-shadow color: ${data.shadowColor}, font size: ${data.fontSize}, font weight: ${data.fontWeight}). Oczekiwany współczynnik kontrastu: ${data.expectedContrastRatio}",
+        "shadowOnBgColor": "Element ma niewystarczający kontrast kolorów: of ${data.contrastRatio} pomiędzy kolorem cienia a kolorem tła (text-shadow color: ${data.shadowColor}, background color: ${data.bgColor}, font size: ${data.fontSize}, font weight: ${data.fontWeight}). Oczekiwany współczynnik kontrastu: ${data.expectedContrastRatio}"
```
Comment (WilcoFiers) [inline locales/pl.json:598]: Should those colons (`:`)  be after "kontrast kolorów"?

**Review by WilcoFiers — CHANGES_REQUESTED** [review]
@zwiastunsw I left a couple minor comments, please have a look and let me know if this is correct.

File: locales/pl.json [hunk 2]
```diff
@@ -2,120 +2,136 @@
...
     "aria-roles": {
-      "description": "Wszystkie elementy z atrybutem roli używają prawidłowej wartości tego atrybutu.",
-      "help": "Stosowane role ARIA muszą być zgodne z obowiązującymi wartościami."
+      "description": "Wartości atrybutu role są poprawne.",
+      "help": "Stosowane role ARIA muszą mieć poprawne wartości."
     },
     "aria-text": {
       "description": "Atrybut \"role=text\" jest używany dla elementów, które nie mają potomków przyjmujących fokus",
-      "help": "\"role=text\" nie powinien mieć potomków przyjmujących fokus."
+      "help": "Element z atrybutem \"role=text\" nie może mieć potomków przyjmujących fokus."
```
Comment (WilcoFiers) [inline locales/pl.json:86]: That should probably be like this. We only recently caught this issue in the original.

```suggestion
      "help": "Element z atrybutem role=\"text\" nie może mieć potomków przyjmujących fokus."
```

**Review by WilcoFiers — CHANGES_REQUESTED** [review]
Two very minor comments. The rest looks excellent.

**Comment by zwiastunsw** [comment]
Thx

**Pull request merged**